prow/cmd/pipeline/main.go

```diff
@@ -156,8 +156,10 @@ func main() {
 			logrus.WithError(err).Infof("Ignoring cluster context %s: tekton pipeline CRD not deployed", context)
 			continue
 		}
+		// Don't panic when a build cluster cannot be reached
 		if err != nil {
-			logrus.WithError(err).Fatalf("Failed to create %s pipeline client", context)
+			logrus.WithError(err).Warningf("Failed to create %s pipeline client", context)
```

Comment:
nit: I think this should be error level rather than warning.

Reply:
Other major prow components already reported this as error, I think it's ok to make it a warning since we'll always get it from other prow components already